src/UI/CardPage.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import Button from "react-bootstrap/Button";
+import Card from "react-bootstrap/Card";
+import styles from "./CardPage.module.css";
+import { CardHomeInterface } from "../interfaces/CardHomeInterface";
+import { Link } from "react-router-dom";
+
+const CardPage = (props: CardHomeInterface) => {
+  return (
+    <div className={styles.card}>
+      <Card
+        className="bg-dark text-white"
+        style={{ width: "12vw", height: "23em" }}
```

Comment:
I always question "magic numbers"... what exactly is 12vh and 23em (specifically, what is the significance of 12 and 23)? These are going to result in different values based on device size and user's font size. Is that what you want? Is this going to look good across a wide range of devices and sizes?

Reply:
I see what you were saying. It will be addressed in the next pull request